PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MD004" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Lǝssāna sabʾ</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p>Digitized within the framework of the PhD project "Scribes and Practices of
+                    Manuscript Production in 21st Century Gondar Area, Ethiopia", under the project
+                    of "Documenting an Ancient Education System in Africa: ʾAbənnat Təmhərt in
+                    Ethiopia" funded by Gerda Henkel Stiftung. </p>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>
+                        <idno>MD004</idno>
+                    </msIdentifier>
+                    <msContents>
+                          <msItem xml:id="ms_i1">
+                           <title type="complete" ref="LIT3034RepCh314"/>
+                            <textLang mainLang="gez"/>
+                              <incipit>በስመ: አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ ልሳነ፡ ምቀኛ፡ ወተንኮለኛ፡ ናሁ፡ ለዚባዎስሙ፡ 
+                                  በዝ፡ ድብ፡ ያዕቆብ፡ እምልሳነ፡ ሰብእ፡ ወነገደ፡ ያዕቆብ፡ ምድረ፡ </incipit>
+                              <explicit><unclear>ባህረ፡ ልሳነ፡ ፍስሐ፡ ገፁ፡ ተማህፀንኩ፡ በጸሎትከ፡ ሞተ፡ አቡ፡ ወዕቀበኒ፡ ዘበልማድ፡ እምዝ፡ ቆንተንቶሙ፡ ብከ፡ እስከተሐዉሱ፡ እምባእድ፡ 
```

Comment:
Is the whole text unclear or only some of the words/letters? It is rare that such a long passage is unreadable.

Reply:
> Is the whole text unclear or only some of the words/letters? It is rare that such a long passage is unreadable.

The last panel is really hard for me to read, at least
![DSC01281](https://github.com/user-attachments/assets/c588900a-e5f6-440a-88a8-687f683db9f5)
.

Reply:
I suggest marking only those letters with `<unclear>` that are not clearly legible and therefore uncertain.